README.md

```diff
@@ -7,18 +7,18 @@ Openliberty.io is a portable, mobile enabled web application hosted on [IBM Clou
 The graphical user interface is built to work consistently across all major web browsers, including Chrome, Edge, Internet Explorer, Firefox and Safari.
 
 ## Responsive design
-The [jQuery](https://jquery.com/) and [Bootstrap](http://getbootstrap.com/) frameworks are leveraged to provide a seamless experience in desktops, laptops, tablets and smart phones.
+The [jQuery](https://jquery.com/) and [Bootstrap](http://getbootstrap.com/) frameworks are leveraged to provide a seamless experience in desktops, laptops, tablets, and smart phones.
 
 ## Contributing to the blog
-Create a pull request with the content of the blog post placed in the `src/main/content/_posts/` folder using the following file naming scheme: `YYYY-MM-DD-post-title.extension`. HTML, markdown and asciidoc formats can be used. The file extension would be html, md or adoc respectively. In the blog post file the following front matter variables must be set:
+Create a pull request with the content of the blog post placed in the `src/main/content/_posts/` folder using the following file naming scheme: `YYYY-MM-DD-post-title.extension`. HTML, markdown, and asciidoc formats can be used. The file extension would be html, md, or adoc respectively. In the blog post file the following front matter variables must be set:
 - layout: post
 - categories: blog
 - title: `title of the blog post`
 - date: `date in YYYY-MM-DD HH:MM:SS +/-TTTT format`
 - author_picture: `secure url to author picture`
 
 ## Contributing to the guides
-Each guide resides in its own repository and is dynamically pulled into the openliberty.io build process through the `scripts/build.sh` shell script. The content of the guide can be written in HTML, markdown or asciidoc formats. The following front matter variables must be set:
+Each guide resides in its own repository and is dynamically pulled into the openliberty.io build process through the `scripts/build.sh` shell script. The content of the guide can be written in HTML, markdown, or asciidoc formats. The following front matter variables must be set:
```

Comment:
To be consistent, write file formats as:
.html, .md, or .adoc 

Alternatively, you can write HTML, markdown, or AsciiDoc